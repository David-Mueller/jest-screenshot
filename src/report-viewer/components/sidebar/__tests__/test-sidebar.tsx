import * as React from "react";
import { shallow } from "enzyme";
import { Sidebar } from "..";

describe("Sidebar", () => {
    it("looks as expected", () => {
        expect(shallow(<Sidebar />)).toMatchSnapshot();
    });
});
