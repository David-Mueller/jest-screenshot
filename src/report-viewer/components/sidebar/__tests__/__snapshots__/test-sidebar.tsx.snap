// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar looks as expected 1`] = `
<aside
  className="column is-3 section is-fullheight menu"
>
  <p
    className="menu-label"
  />
  <FileList />
</aside>
`;
